### ISSUE TITLE
Much bigger than apollo-cache-inmemory

### ISSUE THREAD

**Issue by wmertens** [convoyinc/apollo-cache-hermes#421]
I don't know how, but when I switch apollo-cache-inmemory for apollo-cache-hermes it increases my gzipped bundle size by 14kb.

Webpack-bundle-analyzer shows that inmemory is 23kb parsed, and hermes is 48kb.

🤔

**Comment by nevir** [comment]
Yeah, hermes is definitely more complicated than inmemory (though the upcoming v3 of inmemory may change that) - there are probably some areas where we could trim some fat, though
